airmozilla/main/static/main/js/calendar.js

```diff
@@ -37,7 +42,10 @@ $(function() {
     startsOnMonday.on('change', function(){
         localStorage.setItem('startsOnMonday', startsOnMonday.prop('checked')? 1 : 0);
         container.fullCalendar('destroy');
+        view = localStorage.getItem('view');
         initCalendar();
+        $('.fc-right button').on('click', saveView);
     });
     initCalendar();
+    $('.fc-right button').on('click', saveView);
```

Comment:
So I saw your other comment and it's a fair point but I've look into this in detail and I think we should use [eventAfterAllRender](http://fullcalendar.io/docs/event_rendering/eventAfterAllRender/).
It gets fired every time you load the calendar (after the events in it have been rendered). 
It *also* gets fired when you switch the view. 

(it's almost the same as `viewRender` but happens slightly later)

It sucks that there's no way of distinguishing between initial rendering and manual-view-switching, but what sucks even more is depending on the hack of knowing the exact CSS class of the area where the buttons are. 

Your trick is clever and has advantages but it also means it's a hack and it might stop working at some future revision. It's generally much better to stick to the documented public API. 

So, would you mind changing to that. That means it's writing to the localStorage every time you open the calendar. But I guess so be it.